README.md

```diff
@@ -15,6 +15,12 @@ Read the [AWS docs on EKS to get connected to the k8s dashboard](https://docs.aw
 * You want these resources to exist within security groups that allow communication and coordination. These can be user provided or created within the module.
 * You've created a Virtual Private Cloud (VPC) and subnets where you intend to put the EKS resources. The VPC satisfies [EKS requirements](https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html).
 
+## Important note
+
+The default `cluster_version`is now 1.16. Kubernetes 1.16 includes a number of deprecated API removals, and you need to ensure your applications and add ons are updated, or workloads could fail after the upgrade is complete. For more information on the API removals, see the [Kubernetes blog post](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/). For action you may need to take before upgrading, see the steps in the [EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html#1-16-prequisites).
```

Comment:
git-chglog mangles the second link with the anchor ref. It generates: `[EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[#1](https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1)-16-prequisites)`

I have no idea what to do about that apart from removing `#1-16-prequisites`

Reply:
Ohh nice catch. I removed `#1-16-prequisites`. We'll now have.

BREAKING CHANGES:
- The default `cluster_version`is now 1.16. Kubernetes 1.16 includes a number of deprecated API removals, and you need to ensure your applications and add ons are updated, or workloads could fail after the upgrade is complete. For more information on the API removals, see the [Kubernetes blog post](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/). For action you may need to take before upgrading, see the steps in the [EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html). Please set explicitly your `cluster_version` to an older EKS version untill your workload are ready for Kubernetes 1.16.

Reply:
It looks like the change wasn't pushed?

Reply:
I change it in git log only for git-chglog. The readme still has the full link.

Reply:
Ah. I see 😄